appstore-metadata-service/src/main/resources/static/appstore-metadata-service.yaml

```diff
@@ -733,6 +733,11 @@ components:
               type: boolean
               description: Visible version attribute
               example: true
+            encryption:
+              type: boolean
+              description: Encryption version attribute
```

Comment:
Maybe we could rename to "bundle encryption attribute"? not sure if "version" word is relevant here